R/print-functions.R

```diff
@@ -434,21 +505,35 @@ print.OpenEOCapabilities = function(x, ...) {
 
 #' @export
 print.ResultList = function(x, ...) {
-    cat("Results for job: ",x$id,"\n")
-    if (length(x$properties$expires) > 0) {
-        cat("Links expire on: ",x$properties$expires,"\n")
+    if (is_html_context()) {
+        unclass(x$assets)
+        return(print_html("batch-job-results", x))
```

Comment:
I tried to fix this, but I couldn't get it to work. Maybe there is something wrong with the template call. At least I couldn't visualize this. Do need to visualize the assets or the whole job results?

Reply:
I'll check list_results again, but I think it worked flawlessly for me. What issues are you encountering and at which back-end?